com.blueokiris.bgrm.json

```diff
@@ -0,0 +1,70 @@
+{
+    "app-id": "com.blueokiris.bgrm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
```

Comment:
```suggestion
    "runtime-version": "22.08",
```

Reply:
Upgraded to 22.08

Reply:
In 22.08, setuptools is upgraded, which [causes issues with numpy](https://github.com/numpy/numpy/issues/22157)

I need numpy as a dependency for other things, which means as far as I can tell, my two options are:
- use the numpy .whl which is x86_64 exclusive
- use 21.08 with old setuptools so I can build numpy from source